src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
Pretty easy and allows us to support backwards compatibility with browsers that don't have full support for `input[type=range]`. Thoughts on what should be the default for the reference implementation?